pilot/lib/pilot/build_manager.rb

```diff
@@ -389,15 +389,22 @@ def expire_previous_builds(build)
     def transporter_for_selected_team(options)
       # Use JWT auth
       api_token = Spaceship::ConnectAPI.token
+      api_key = if options[:api_key].nil? && !options[:api_key_path].nil?
+                  # Load api key info if user set api_key_path, not api_key
+                  { key_id: api_token.key_id, issuer_id: api_token.issuer_id, key: api_token.key_raw, is_key_content_base64: api_token.is_key_content_base64 }
```

Comment:
Same question as above regarding testing `options[:api_key_path].nil?` instead of `api_token.nil?`